style.css

```diff
@@ -140,3 +151,297 @@ body {
 .link a:active img {
   filter: saturate(100);
 }
+
+/* Works Section */
+#works-section {
+  display: grid;
+  grid: 'header' 'card1' 'card2' 'card3' 'card4' 'card5' 'card6' 'card7';
+  grid-gap: 5vw;
+  padding: 15vw 10vw;
+}
+
+#works-header h2 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #1c1a19;
+}
+
+#header-line {
+  border-top: 1px solid #212121;
+}
+
+/* Cards */
+.tag-group {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.card1 h3,
+.card1 p,
+.card1 .tag-group {
+  color: #091e42;
+}
+
+.content h3 {
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 22px 0;
+}
+
+.content p {
+  font-family: 'Inter', sans-serif;
+  margin: 0;
+}
+
+.tag {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px;
+  margin: 8px 12px 8px 0;
+}
+
+.card1 .tag {
+  border: 1px solid #8993a4;
+}
+
+.main-btn {
+  padding: 10px;
+  margin-top: 10px;
+}
+
+.btn {
+  display: inline-block;
+  text-decoration: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: #ff6b00;
+  border: 1px solid #ff6b00;
+  color: #fff;
+  transition: all 0.2s;
+}
+
+.btn:visited {
+  color: #5b5957;
+  background-color: #f4f1f0;
+  border: 1px solid #f4f1f0;
+}
+
+.btn:hover {
+  color: #f55800;
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+}
+
+.btn:active {
+  color: #f4f1f0;
+  background-color: #e05304;
+  border: 1px solid #e05304;
+  transition: all 0s;
+}
+
+/* Cars2 */
+.card {
+  background-image: url("images/cards/card2.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+}
```

Comment:
- [ ] You did a really good job implementing these two sections, but one of the requirements is to use CSS flexbox to align the card's content, so please make sure to apply the necessary changes to your project.